library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js

```diff
@@ -22,34 +22,40 @@
 
         portal.i18n.translations[options.namespace] = portal.i18n.translations[options.namespace] || {};
 
-        $PBJQ.ajax({
-            url: '/sakai-ws/rest/i18n/getI18nProperties',
-            cache: true,
-            dataType: "text",
-            data: {locale: portal.locale,
-                    resourceclass: options.resourceClass,
-                    resourcebundle: options.resourceBundle},
-            })
-            .done(function (data, textStatus, jqXHR) {
-
-                data.split("\n").forEach(function (pair) {
-
-                    var keyValue = pair.split('=');
-                    if (keyValue.length == 2) {
-                        portal.i18n.translations[options.namespace][keyValue[0]] = keyValue[1];
-                    }
-
-                    if (options.debug) {
-                        console.log('Updated translations: ');
-                        console.log(portal.i18n.translations[options.namespace]);
-                    }
+        if (sessionStorage.getItem(portal.locale + options.resourceClass + options.resourceBundle) !== null) {
+            portal.i18n.translations[options.namespace] = JSON.parse(sessionStorage[portal.locale + options.resourceClass + options.resourceBundle]);
+            if (options.callback) options.callback();
```

Comment:
braces here

Reply:
well, this was in this way on the original :D .. I'm going to modify it ASAP